<?xml version="1.0" ?>
<sdf version="1.5">
<model name="unit_cylinder_1">
  <pose>0.0 3.0 1.38235 0.0 0.0 0.0</pose>
  <static>1</static>
  <link name="link">
    <kinematic>0</kinematic>
    <inertial>
      <mass>1</mass>
      <inertia>
	<ixz>0</ixz>
	<ixx>1</ixx>
	<ixy>0</ixy>
	<izz>1</izz>
	<iyy>1</iyy>
	<iyz>0</iyz>
      </inertia>
    </inertial>
    <self_collide>0</self_collide>
    <gravity>1</gravity>
    <collision name="collision">
      <geometry>
	<cylinder>
	  <length>1.38235</length>
	  <radius>0.0456111</radius>
      </cylinder>
      </geometry>
      <max_contacts>10</max_contacts>
      <surface>
	<contact>
	  <ode></ode>
	</contact>
	<friction>
	  <ode></ode>
	</friction>
	<bounce></bounce>
      </surface>
    </collision>
    <visual name="visual">
      <geometry>
	<cylinder>
	  <length>1.38235</length>
	  <radius>0.0456111</radius>
	</cylinder>
      </geometry>
      <material>
	<script>
	  <uri>file://media/materials/scripts/gazebo.material</uri>
	  <name>Gazebo/Grey</name>
	</script>
      </material>
    </visual>
    <velocity_decay>
      <linear>0</linear>
      <angular>0</angular>
    </velocity_decay>
  </link>
</model>
</sdf>
